org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/ILanguageServerInfoProvider.java

```diff
@@ -0,0 +1,15 @@
+package org.eclipse.jdt.ls.debug.adapter;
+
+public interface ILanguageServerInfoProvider {
+    String getFullyQualifiedName(String sourceFilePath, int line, int column);
```

Comment:
How about the caller to do the batch operation as each line only resolve one fullyqualifiedname?